source/uwp/Renderer/lib/XamlHelpers.cpp

```diff
@@ -1229,4 +1256,24 @@ namespace AdaptiveCards::Rendering::Uwp::XamlHelpers
 
         return S_OK;
     }
+
+    winrt::Windows::UI::Xaml::UIElement HandleInputLayoutAndValidation(winrt::AdaptiveCards::ObjectModel::Uwp::IAdaptiveInputElement adaptiveInput,
+                                                                       winrt::Windows::UI::Xaml::UIElement inputUIElement,
+                                                                       boolean hasTypeSpecificValidation,
+                                                                       winrt::AdaptiveCards::Rendering::Uwp::IAdaptiveRenderContext renderContext)
+    {
+        ComPtr<IUIElement> inputLayout;
```

Comment:
should this be `winrt::com_ptr`?

Reply:
Yes, i was making this unnecessarily complicated.